docs/classic-ui/page-templates/dtml.md

```diff
@@ -0,0 +1,7 @@
+# DTML
+
+DTML technology has been phased out many years ago.
```

Comment:
What does `DTML` mean? Always use the full name of an acronym when first introduced, followed by its acronym in parentheses.

```
Full Name of Thing (FNoT)
```

Also I added a Glossary in my open PR. Terms should always be included and defined in the glossary.